pkg/bundle/controller.go

```diff
@@ -46,8 +46,11 @@ import (
 )
 
 func CacheOpts(opts controller.Options) cache.Options {
+	ctrCacheNamespaces := setupCacheNamespaces(append([]string{opts.Namespace}, opts.TargetNamespaces...)...)
```

Comment:
```suggestion
	ctrCacheNamespaces := setupCacheNamespaces(opts.Namespace, opts.TargetNamespaces...)
```

I think this should work and is simpler to read.

Reply:
Ok, TIL! Go has some sharp corners. Oh well....

Reply:
```suggestion
	ctrCacheNamespaces := setupCacheNamespaces(append(opts.TargetNamespaces, opts.Namespace)...)
```

But this works, and is a little easier to read.